src/metabase/api/database.clj

```diff
@@ -1042,6 +1043,18 @@
                                       [:in :collection_id (api/check-404 (seq (db/select-ids Collection :name schema)))])])
          (map api.table/card->virtual-table))))
 
+(api/defendpoint GET "/:id/models"
+  "Returns a list of models for a database."
+  [id]
+  {id su/IntGreaterThanZero}
+  (api/read-check Database id)
+  (prn "GET /api/database/:id/models" id)
+  (let [cards (-> (db/select Card, :database_id id, :archived false, :dataset true, {:order-by [[:%lower.name :asc]]})
+                  (hydrate :creator :collection))
```

Comment:
do we use information about `creator` on the FE? I don't see any from your demo on loom.

Reply:
No, but I figured I'd mimic the "/api/card" endpoint and provide it. Since these are card objects, I wasn't sure whether they would be cached on the FE by ID and type, and then some other component would find it missing. @ranquild Can you answer that question?